### PR TITLE
Update MKO

### LINKED ISSUE #311: Update Knap04 JHK photometry band
All photometry from Knapp et al 2004 is currently using MKO.JHK and should be UKIRT/UFTI.JHK.  Related to #301.

### PR DESCRIPTION
**Pull request by SherelynA**
Related to #301 
Closes #311

- [ ] MKO Photometry

### CODE REVIEW

File: scripts/Update_Photometry.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    where(and_(db.Photometry.c.band == 'MKO.J', db.Photometry.c.reference == 'Best20b')). \
+    values(band='UKIRT/WFCAM.J').execute()
+
+db.Photometry.update(). \
+    where(and_(db.Photometry.c.band == 'MKO.Y', db.Photometry.c.reference == 'Best20b')). \
+    values(band='UKIRT/WFCAM.Y').execute()
+
+db.Photometry.update(). \
+    where(and_(db.Photometry.c.band == 'MKO.K', db.Photometry.c.reference == 'Best20b')). \
+    values(band='UKIRT/WFCAM.K').execute()
```
Comment (Will-Cooper) [inline scripts/Update_Photometry.py:134]:
```suggestion
mkod = {f'MKO.{band}': f'UKIRT/WFCAM.{band}' for band in ('Y', 'J', 'H', 'K')}
for ref in ('Best20a', 'Best20b'):
    for mkoband, ukirtband in mkod.items():
        db.Photometry.update().where(and_(db.Photometry.c.band == mkoband, db.Photometry.c.reference == ref)).values(band=ukirtband).execute()
```

**Review by Will-Cooper — COMMENTED** [review]
try this

**Review by Will-Cooper — COMMENTED** [review]
Looking good!

**Comment by SherelynA** [comment]
Made too many mistakes with creating JSON files corresponding to the wrong filters name so I will start this over in a new pull request V2.

**Comment by SherelynA** [comment]
Starting over in #352

**Pull request closed without merging**